### PR TITLE
Add contribution guide

### LINKED ISSUE #95: Add contribution guide
We should add a contribution guide (`CONTRIBUTING.md`) to help guide folks submitting issues and PRs against this project.

We should also add a link to the contribution guide in the project README.

### PR DESCRIPTION
**Pull request by devillexio**
This PR introduces a guide to contributing to the project.

Resolves #95.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+## How to contribute
+
+Thank you for taking the time to contribute to the Helm chart for the 1Password SCIM bridge.
+
+#### **Did you find a bug?**
+
+* Do not open up a GitHub issue if the bug is an issue with the 1Password SCIM bridge or functionality related to automated provisioning, and instead reach out to the our [support team](https://support.1password.com/contact/) directly.
```
Comment (grellyd) [inline CONTRIBUTING.md:7]:
```suggestion
* If the bug an issue with the 1Password SCIM bridge or functionality related to automated provisioning in 1Password, do not open up a GitHub issue. Instead reach out to the our [support team](https://support.1password.com/contact/) directly.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+* Ensure the bug was not already reported by searching on GitHub under [Issues](https://github.com/1Password/op-scim-helm/issues).
```
Comment (grellyd) [inline CONTRIBUTING.md:9]:
```suggestion
* Search if the bug has already been reported by searching on GitHub under [Issues](https://github.com/1Password/op-scim-helm/issues). Comment on the issues with your context to help us resolve the issue.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+## How to contribute
+
+Thank you for taking the time to contribute to the Helm chart for the 1Password SCIM bridge.
+
+#### **Did you find a bug?**
+
```
Comment (grellyd) [inline CONTRIBUTING.md:6]: Perhaps these should all be an ordered list, instead of unordered.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+#### **Did you write a patch that fixes a bug?**
+
+* Open a new GitHub pull request with the patch.
+
+* Ensure the PR description clearly describes the problem and solution. Include the relevant issue number if applicable.
+
+#### **Did you fix whitespace, formatting, or make a purely cosmetic patch?**
+
+Changes that are cosmetic in nature and do not add anything substantial to the stability, functionality, or testability of this project will generally not be accepted.
```
Comment (grellyd) [inline CONTRIBUTING.md:21]:
```suggestion
Changes that are cosmetic in nature and do not add anything substantial to the stability, functionality, or testability of this project will be rejected.
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+* Find more information on our [support portal](https://support.1password.com/scim/).
+
+* Ask a question on our [community support forum](https://1password.community/categories/scim-bridge).
+
+* Contact our [support team](https://support.1password.com/contact/).
+
+Thanks! :heart:
+
+Provisioning Team @ [1Password](https://1password.com)
```
Comment (grellyd) [inline CONTRIBUTING.md:41]:
```suggestion
The Provisioning Team @ [1Password](https://1password.com)
```

**Review by grellyd — APPROVED** [review]
Some minor phrasing suggestions, but overall looks great! Thanks DV.

File: CONTRIBUTING.md (hunk 3 above)
Comment (devillexio) [reply CONTRIBUTING.md:6]: I went back and forth on whether these lists should be ordered or unordered and ultimately kept it unordered for consistency.

Also, I quite like the way the Rails team did [theirs](https://github.com/rails/rails/blob/main/CONTRIBUTING.md), which I used as a starting point for this.

**Review by grellyd — APPROVED** [review]
Merge away my friend.

**Review by alicethorne-ab — APPROVED** [review]
Looks good to me!

**Pull request merged**